data/json/npcs/robofac/robofac_basement_npcs/NPC_Darla_Novak.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "type": "npc",
+    "id": "HUB01_DNovak",
+    "name_unique": "Darla Novak",
+    "gender": "female",
+    "name_suffix": "building maintenance",
+    "class": "NC_HUB01_DNovak",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "HUB01_DNovak_Intro",
+    "faction": "robofac"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_HUB01_DNovak",
+    "name": { "str": "Hub 01 HVAC maintenance" },
+    "job_description": "I'm part of Hub 01's maintenance crew.",
+    "common": false,
+    "//": "HVAC maintenance.  Keeps melchior from boiling to death and you from choking on stale bunker air.",
+    "bonus_str": { "rng": [ -1, 2 ] },
+    "bonus_dex": { "rng": [ -1, 2 ] },
+    "bonus_int": { "rng": [ -2, 2 ] },
+    "bonus_per": { "rng": [ -1, 2 ] },
+    "worn_override": "HUB01_DNovak_worn",
+    "carry_override": "HUB01_DNovak_carried",
+    "weapon_override": "HUB01_DNovak_wield",
+    "traits": [ { "group": "Appearance_demographics" } ],
```

Comment:
Does Appearance Demographics check gender above?

Reply:
Hmm good question, let me check

Reply:
So it doesnt include beards or anything so I guess it does.